### ISSUE TITLE
Could you share a Readme in English

### ISSUE THREAD

**Issue by zhangsanfeng86** [SMART-TTS/SMART-Single_Emotional_TTS#3]
Hi, thanks for  your share!

Could you share a Readme in English?

**Comment by SMART-TTS** [comment]
Hello!
I have translated the readme script up until [To do] part below. Hope this helps.

This repo is a Transformer-TTS based single speaker emotional TTS, which is a part of SMART-TTS project. This source code is released as a part of a project financially funded by IITP. 
SMART-TTS_Singl_Emotional model is an unsupervised style TTS model which is based on Transformer-TTS with a variable-length style embedding extracted from reference.
We have trained with an Korean emotional DB which is privately used within our lab for only research purpose. You could download the pre-trained model (in the link).

There are some improvements to the vanilla Transformer-TTS
- Improvement in attention failure using monotonic interpolation
- Improvement in attention failure using sliding window attention masking, replacing stop token
- Global & local style embedding extracted by variable-length style embedding
- Style conditioning using scaled dot-product attention
- Additional STFT loss
- High quality synthesis with SMART-Vocoder
- Length regulation with linear interpolation on mel-spectrograms
